### PR TITLE
Fix links and header case in key-event-migration guide

### PR DESCRIPTION
**Pull request by parlough**

### CODE REVIEW

File: src/release/breaking-changes/key-event-migration.md [hunk 1]
```diff
@@ -18,12 +18,14 @@ In the original key event subsystem, handling each platform's quirks in the
 framework and in client apps caused overly complex code, and the old system
 didn't properly represent the true state of key events on the system.
 
-[RawKeyboard] is the legacy API, and has been deprecated and will be removed in
-the future. The replacements are [HardwareKeyboard] and [KeyEvent] APIs (such as
-[FocusNode.onKeyEvent]).
+[`RawKeyboard`][] is the legacy API, and has been deprecated and
+will be removed in the future.
+The replacements are [`HardwareKeyboard`][] and [`KeyEvent`][] APIs
+(such as [`FocusNode.onKeyEvent`][]).
```
Comment (atsansone) [inline src/release/breaking-changes/key-event-migration.md:24]: **chore:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
The legacy API [`RawKeyboard`][] has been deprecated and will be removed in the future.
The [`HardwareKeyboard`][] and [`KeyEvent`][] APIs replace this legacy API.
An example would be [`FocusNode.onKeyEvent`][].
```

File: src/release/breaking-changes/key-event-migration.md [hunk 2]
```diff
@@ -18,12 +18,14 @@ In the original key event subsystem, handling each platform's quirks in the
...
 
-Behavior-wise, [RawKeyboard] provided a less unified, less regular event model
-than [HardwareKeyboard] does. For example:
+Behavior-wise, [`RawKeyboard`][] provided a
+less unified,less regular event model than [`HardwareKeyboard`][] does.
+For example:
```
Comment (atsansone) [inline src/release/breaking-changes/key-event-migration.md:28]: **chore:** Don't make up words.
```suggestion
The behavior of [`RawKeyboard`][] provided a less unified and less regular event model
than [`HardwareKeyboard`][] does. Consider the following examples.
```

File: src/release/breaking-changes/key-event-migration.md [hunk 3]
```diff
@@ -87,8 +89,9 @@ their functionality is no longer offered.
 
 ## Migration guide
 
-APIs provided by the Flutter framework are already migrated. Migration is needed
-only if any of the classes or methods listed in the previous section are used.
+APIs provided by the Flutter framework are already migrated.
+Migration is needed only if any of the classes or methods listed in
+the previous section are used.
```
Comment (atsansone) [inline src/release/breaking-changes/key-event-migration.md:94]: **chore:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
Flutter framework API have been migrated.
If your code includes any of the classes or methods listed in
the previous section, migrate to these new APIs.
```

File: src/release/breaking-changes/key-event-migration.md [hunk 4]
```diff
@@ -208,10 +211,11 @@ KeyEventResult _handleKeyEvent(KeyEvent _) {
...
 
-Be careful to check conditionals: a [`KeyRepeatEvent`][] is also a key down event,
-but it is a different type (it is not a subclass of [`KeyDownEvent`][]), so
-don't assume that `keyEvent is! KeyDownEvent` only allows key up events, because
-both `KeyDownEvent` and `KeyRepeatEvent` need to be checked.
+Be careful to check conditionals:
+a [`KeyRepeatEvent`][] is also a key down event, but
+it is a different type (it is not a subclass of [`KeyDownEvent`][]), so
+don't assume that `keyEvent is! KeyDownEvent` only allows key up events,
+because both `KeyDownEvent` and `KeyRepeatEvent` need to be checked.
```
Comment (atsansone) [inline src/release/breaking-changes/key-event-migration.md:218]: **issue:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).

```suggestion
Be careful to check conditionals. 
Though not inheriting from [`KeyDownEvent`][],
a [`KeyRepeatEvent`][] event works like a key down event.
Don't assume that `keyEvent is! KeyDownEvent` only allows key up events.
Check both `KeyDownEvent` and `KeyRepeatEvent` events.
```

**Review by atsansone — APPROVED** [review]
LGTM if updated for active voice and readability.

**Comment by parlough** [comment]
Thanks for the review @atsansone! I didn't update the text originally, but I'm happy to make some adjustments while I'm here.

I applied your suggestions with a few modifications.

**Pull request merged**